grunt-ngdocs-index.tmpl

```diff
@@ -31,7 +31,7 @@
       }
 
       addTag('base', {href: baseUrl});
-      addTag('link', {rel: 'stylesheet', href: 'css/prettify.css', type: 'text/css'});
+      // addTag('link', {rel: 'stylesheet', href: 'css/prettify.css', type: 'text/css'});
```

Comment:
I left it incase we noticed that parts of the page (not patternfly components) didn't look right, we would have the hint as to where settings might be.